exercises/sam_jenkins/ex02_oo_basics/src/Square.h

```diff
@@ -0,0 +1,17 @@
+#ifndef GUARD_Square_h
+#define GUARD_Square_h
+#include "Shapes.h"
+class Square : public Shapes {
+public:
+	Square();
+	Square(double side);
+	double calcPerimiter()const;
+	double calcArea()const;
+private:
+	double side;
```

Comment:
We don't need to side member variable here. We can just use `getSides` from the superclass.